tests/e2e/e2e_common.go

```diff
@@ -178,3 +181,12 @@ func setClusterFlavor(clusterFlavor cnstypes.CnsClusterFlavor) {
 		vanillaCluster = true
 	}
 }
+
+// isValueSet check whether the environment variable is set or not
+func isValueSet(varName string) bool {
```

Comment:
where is this used? Also, `isEnvVarSet()` might be a better name

Reply:
it is used in VVOL and NFS datastores test cases. As of now our pipelines do not run on these datastores and hence we do not set this value in export variable.  If it is set the test will run , otherwise it will skip the test case 

isEnvVariableSet := isValueSet(envSharedVVOLDatastoreURL)
		if !isEnvVariableSet {
			ginkgo.Skip("Skipping Verify online volume expansion on shared VVOL datastore - Because VVOL datastore URL is not set")
		} else {